code/ship/shipfx.cpp

```diff
@@ -2526,7 +2527,7 @@ void engine_wash_ship_process(ship *shipp)
 	float dist_sqr, inset_depth, dot_to_ship, max_ship_intensity;
 	polymodel *pm;
 
-	float max_wash_dist, half_angle, radius_mult;
+	float max_wash_dist, half_angle, radius_mult, wash_intensity;
```

Comment:
If new variables are introduced, please place them as near to their usage as possible.

Reply:
Will do!